projects/storefrontlib/src/cms-components/configurator/commons/config-product-title/config-product-title.component.html

```diff
@@ -0,0 +1,69 @@
+<ng-container *ngIf="product$ | async as product">
+  <div class="cx-config-container">
+    <div class="cx-config-product-title">
+      {{ product.name }}
+    </div>
+
+    <a
+      class="cx-config-toogle-details-link"
+      role="link"
+      (click)="click()"
+      tabindex="0"
+    >
+      <ng-container *ngIf="!showMore"
+        ><div class="cx-config-toogle-details-link-text">
+          {{ 'configurator.header.showMore' | cxTranslate }}
+        </div>
+        <cx-icon [type]="iconTypes.CARET_DOWN"></cx-icon
+      ></ng-container>
+
+      <ng-container *ngIf="showMore"
+        ><div class="cx-config-toogle-details-link-text">
+          {{ 'configurator.header.showLess' | cxTranslate }}
+        </div>
+        <cx-icon [type]="iconTypes.CARET_UP"></cx-icon
+      ></ng-container>
+    </a>
+
+    <div class="cx-config-product-title-details" [class.open]="showMore">
+      <div class="cx-config-product-title-details-image">
+        <img
+          [alt]="getProductImageAlt(product)"
+          [src]="getProductImageURL(product)"
+        />
+      </div>
+      <div class="cx-config-product-title-details-content">
+        <div class="cx-config-product-detail-title">
+          {{ product.name }}
+        </div>
+        <div class="cx-config-product-code">
+          {{ product.code }}
+        </div>
+
+        <div
+          *ngIf="
+            product.priceRange &&
+            (product.priceRange?.maxPrice || product.priceRange?.minPrice)
+          "
+          class="cx-config-product-price"
+        >
+          {{ product.priceRange?.minPrice?.formattedValue }} -
+          {{ product.priceRange?.maxPrice?.formattedValue }}
+        </div>
+        <div
+          *ngIf="
+            !product.priceRange ||
+            (!product.priceRange?.maxPrice && !product.priceRange?.minPrice)
+          "
+          class="cx-config-product-price"
+        >
+          {{ product.price?.formattedValue }}
```

Comment:
What price should be displayed here? Base price or total price?
At the moment only **dash** is displayed.